Download_ERA5.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-

```

Comment:
Can you please add the license heading and a description of what this file does at the top?

Reply:
This file is the same as the ERA5_downlaoder.py. we can just delete it.